DeepSpeech.py

```diff
@@ -1743,12 +1797,22 @@ def export():
                 input_graph_def=session.graph_def,
                 input_saver_def=saver.as_saver_def(),
                 input_checkpoint=checkpoint_path,
-                output_node_names=','.join(node.op.name for node in six.itervalues(outputs)),
+                output_node_names='logits,initialize_state',
                 restore_op_name=None,
                 filename_tensor_name=None,
                 output_graph=output_graph_path,
                 clear_devices=False,
-                initializer_nodes='')
+                initializer_nodes='',
+                variable_names_blacklist='previous_state_c,previous_state_h')
+
+            # Load and export as string
```

Comment:
Why do we want to export it as pb and pbtxt ?

Reply:
I was going to remove that but forgot. It's there so I can easily load an exported graph into Tensorboard to visualize it.

Reply:
Right. Do we need this by default ? I fear it's going to just waste disk space and time for most of the usecase. Would an extra CLI flag defaulting to false be a good deal ?

Reply:
I'll make it a standalone tool in `bin/graph_binary_to_text.py`